srcpkgs/keystone/patches/fix-lib-name.patch

```diff
@@ -0,0 +1,11 @@
+--- bindings/python/keystone/keystone.py.orig
++++ bindings/python/keystone/keystone.py
+@@ -20,7 +20,7 @@
+     __file__ = inspect.getfile(inspect.currentframe())
+ 
+ _lib_path = split(__file__)[0]
+-_all_libs = ('keystone.dll', 'libkeystone.so', 'libkeystone.dylib')
++_all_libs = ('keystone.dll', 'libkeystone.so.0', 'libkeystone.dylib')
```

Comment:
I've submitted a patch to upstream, looking to see what they say. That patch is better, so we'll see.